ArchiSteamFarm/Steam/Integration/ArchiWebHandler.cs

```diff
@@ -2672,6 +2723,23 @@ public sealed class ArchiWebHandler : IDisposable {
 					return (false, null);
 			}
 		}
+		private async Task<(bool Success, string? Result)> ResolveAccessToken() {
+			if (Bot.IsAccountLimited) {
+				// API key is permanently unavailable for limited accounts
+				return (true, null);
+			}
+
+			Uri request = new(SteamStoreURL, "/pointssummary/ajaxgetasyncconfig");
+
+			ObjectResponse<AccessTokenResponse>? response = await UrlGetToJsonObjectWithSession<AccessTokenResponse>(request).ConfigureAwait(false);
+
+			if (response == null || string.IsNullOrEmpty(response.Content.AccessTokenData?.Webapi_token)) {
```

Comment:
Remove the conditional `?` access here once you fix `AccessTokenData` to be non-null.